### ISSUE TITLE
Make team doors automatically close if you Sprint+Open them

### ISSUE THREAD

**Issue by LoneWolfHT** [MT-CTF/capturetheflag#1073]

**Comment by LoneWolfHT** [comment]
The idea with Sprint + Open is that you're sprinting, so the door can close again relatively quickly, but not in your face. Players used to regular doors would also be less likely to be caught off guard by the new behavior

**Comment by farooqkz** [comment]
I agree players are used to regular doors. But I don't agree because of that we must not implement improvements. In the beginning of CTFv3 most players were used to CTFv2 and many suggested having two servers: one for v2 and the other for v3.

We can have new doors which implement the new behavior and make them replace regular doors slowly.

**Comment by farooqkz** [comment]
@LoneWolfHT Any new thought regarding this?
My opinion is almost as it was before. But I should add that I believe for a majority of players, this feature won't be useful. As they are not sprinting while running out of the base.

**Comment by LoneWolfHT** [comment]
Can try listing the different implementations in a poll

**Comment by farooqkz** [comment]
I think this idea is not good. JMA currently has auto close option and players can enable/disable it with settings. We have a PR for this too: https://github.com/MT-CTF/capturetheflag/pull/1151
